app/views/tag/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Post List</h1>
+<h4><%= @tag = Tag.find(params[:id]) %></h4>
```

Comment:
The girls continued to work on the search controller and on the tags index in the same branch due to inertia :). I guess we worked on one branch for too long. I helped them remove the code for the tags index and the post search and put it in other branches, [implement-tags-part2](https://github.com/RailsGEnthusiasts/RubyGameDev.com/tree/implement-tags-part2) and [search-form](https://github.com/RailsGEnthusiasts/RubyGameDev.com/tree/search-form).

Some cleanup was in order and I'm sure we'll have conflicts later on, but hey, at least they'll get some experience handling difficult merges :D.

We did get the tag index working, though, so I think we'll be able to have the "part 2" pull request ready soon. Do let us know about any other changes to this PR.